daemon/info.go

```diff
@@ -251,6 +252,14 @@ func operatingSystem() string {
 	return operatingSystem
 }
 
+func osVersion() (version string) {
```

Comment:
Discussing with @cpuguy83 - we should probably store the value, so that we don't have to actually fetch it each time we call the `/info` endpoint (in order to make it not more heavy)

Also; we might want to add it to the prometheus metrics